jobs/rocoto/postsnd.sh

```diff
@@ -1,24 +1,26 @@
 #! /usr/bin/env bash
 
 source "${HOMEgfs}/ush/preamble.sh"
-
-###############################################################
-# Source FV3GFS workflow modules
-
-. ${HOMEgfs}/ush/load_fv3gfs_modules.sh
+source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
-if [[ ${status} -ne 0 ]]; then
-    exit "${status}"
-fi
+if (( status != 0 )); then exit "${status}"; fi
```

Comment:
I think we are moving toward making these multi-line if blocks for improved readability.  Also, I prefer `[[ ]]` rather than `(( ))` logical checks as logging will skip over arithmetic operations in parentheses:

```suggestion
if [[ ${status} -ne 0 ]]; then
    exit "${status}"
fi
```